includes/admin/tasks/class-wc-payments-task-disputes.php

```diff
@@ -125,25 +242,80 @@ public function is_complete() {
 	 * @return bool
 	 */
 	public function can_view() {
-		return $this->get_disputes_awaiting_response_count() > 0;
+		return count( $this->get_disputes_needing_response_within_days( 7 ) ) > 0;
 	}
 
 	/**
-	 * Gets the number of disputes which need a response.
+	 * Get disputes needing response within the given number of days.
 	 *
-	 * Because this task is initialized before WC Payments, we can only fetch from the cache (via "get()").
-	 * The dispute status cache cannot be populated via this task. If this value hasn't been cached yet, this task won't show until it is.
+	 * @param int $num_days Number of days in the future to check for disputes needing response.
 	 *
-	 * @return int The number of disputes which need a response.
+	 * @return array Array of disputes needing response within the given number of days.
```

Comment:
Unnecessary duplication.
```suggestion
	 * @return array Disputes needing response within the given number of days.
```